mmcv/ops/csrc/common/cuda/nms_rotated_cuda.cuh

```diff
@@ -43,18 +43,16 @@ __global__ void nms_rotated_cuda_kernel(const int n_boxes,
     // (x_center, y_center, width, height, angle_degrees) here.
     __shared__ T block_boxes[threadsPerBlock * 5];
     if (threadIdx.x < col_size) {
-      block_boxes[threadIdx.x * 6 + 0] =
+      block_boxes[threadIdx.x * 5 + 0] =
           dev_boxes[(threadsPerBlock * col_start + threadIdx.x) * 6 + 0];
```

Comment:
```suggestion
          dev_boxes[(threadsPerBlock * col_start + threadIdx.x) * 5 + 0];
```

Reply:
Should it be 5 rather than 6?

Reply:
with multi_label == 1 there shoud be 6 elements: [x, y, w, h, angle, classid]